### PR TITLE
Add Division changes

### PR DESCRIPTION
**Pull request by virajvchaudhari**

### CODE REVIEW

File: notebook2.ipynb [hunk 1]
```diff
@@ -62,6 +62,19 @@
     "\n",
```
Comment (testAcc123-vir) [inline notebook2.ipynb:62]: <p>Add comment on this as well. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/testAcc123-vir/reviewNBCheck/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (testAcc123-vir) [inline notebook2.ipynb:62]: <p>Please add comment on top of the method. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/testAcc123-vir/reviewNBCheck/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (virajvchaudhari) [reply notebook2.ipynb:62]: Will do and update.

Comment (virajvchaudhari) [reply notebook2.ipynb:62]: <p>Will do and update. </p>

**Review by testAcc123-vir — APPROVED** [review]

**Pull request merged**